src/main/java/org/rocstreaming/roctoolkit/RocReceiver.java

```diff
@@ -179,7 +183,12 @@ public class RocReceiver extends NativeObject {
     private static long tryOpen(RocContext context, RocReceiverConfig config) throws IllegalArgumentException, Exception {
         Check.notNull(context, "context");
         Check.notNull(config, "config");
-        return open(context.getPtr(), config);
+
+        LOGGER.log(Level.FINE, "roc_receiver opening, context ptr=" + context.getHexPtr());
+        long ptr = open(context.getPtr(), config);
+        LOGGER.log(Level.FINE, "roc_receiver opened, context ptr={0}, ptr={1}",
+                new Object[]{context.getHexPtr(), Long.toHexString(ptr)});
```

Comment:
Consider unifying pointer formatting to something like:

```
Long.toHexString(context.getPtr()), Long.toHexString(ptr)
```

or:

```
toHex(context.getPtr()), toHex(ptr)
```

(where toHex is NativeObject method)

to make it clear that all pointers are formatted the same way.

Feel free to keep it as is if you think it's OK now.